jdaviz/core/helpers.py

```diff
@@ -37,16 +37,19 @@ class ConfigHelper(HubListener):
         default configuration for this helper.
 
     verbosity : {'debug', 'info', 'warning', 'error'}
-        Verbosity of the application.
+        Verbosity of the popup messages in the application.
+    history_verbosity : {'debug', 'info', 'warning', 'error'}
+        Verbosity of the history logger in the application
     """
     _default_configuration = 'default'
 
-    def __init__(self, app=None, verbosity='info'):
+    def __init__(self, app=None, verbosity='info', history_verbosity='info'):
```

Comment:
What about the CLI and "standalone app" notebooks? See #756

Reply:
Now that we have `history_verbosity` that is less intrusive, is it too controversial to change the default of `verbosity` from `'info'` to `'warning'` too controversial?

Reply:
I'd be in favor of switching the default to `'warning'`, but maybe that is something we should bring up to see if anyone has objections first?

re: voila access - I'm all in favor if everyone is happy with these names and once we settle on the defaults.  Do you think that hook should go in this PR?

Reply:
I guess it can be a follow up PR but it would have to be done eventually so the API is consistent throughout.